src/components/projects/ AirQuality.jsx

```diff
@@ -1,5 +1,83 @@
+import React from "react";
+import Subtitle from "../Subtitle";
 const AirQuality = () => {
-  return <div> AirQuality</div>;
+  return (
+    <div className="grid grid-cols-3 gap-10 m-28 justify-around ">
+      <div className="col-span-2 pb-4 w-5/6 ">
+        <Subtitle title="General Overview"></Subtitle>
+        <p className="ml-2 mt-1">
+          Air quality has overtime deteriorated due to developing countries
+          industrialization. This has in turn put the health of population in
+          risk of respiratory disorders, cardiovascular diseases and
+          asthma/allergy. This project focuses on creating a micro controlled
+          gas sensor system aimed at alerting the user when dangerous levels of
+          pollutants are in the air. We are also developing an automated air
+          filtration system to decrease exposure from harmful particulates in
+          the air.
+        </p>
+      </div>
+      <div>
```

Comment:
you probably don't need this parent div

Reply:
The images don't appear in the right spots unless I make a separate div for them. Do you have some method in mind I could use to implement them correctly without making a separate div for them?